src/ChannelAdam.Soap/Soap12Maker.cs

```diff
@@ -134,53 +134,53 @@ public static XElement CreateSoapBodyWithFault(Soap12FaultCode code, XNamespace?
 
         #region CreateSoapFault
 
-        public static XElement CreateSoapFault(Soap12FaultCode code, string reason)
+        public static XElement CreateSoapFault(Soap12FaultCode code, string reason, string? prefix = null)
```

Comment:
Don't we need to do this above - in CreateSoapEnvelopeWithFault() and CreateSoapBodyWithFault() - passing the prefix into both CreateSoapBody() and CreateSoapFault()?

Reply:
Yep, seems like I forgot to add the passing parameter into CreateSoapFault.